### PR TITLE
check patch apk permission

### PR DESCRIPTION
**Pull request by peacepassion**
close #110

### CODE REVIEW

**Comment by JackCho** [comment]
👍

File: amigo-lib/src/main/java/me/ele/amigo/utils/PermissionChecker.java [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+                return true;
+            }
+            if (hostPs == null) {
+                extraPermissions.addAll(Arrays.asList(patchPs));
+            } else {
+                for (String patchP : patchPs) {
+                    if (!contains(patchP, hostPs)) {
+                        extraPermissions.add(patchP);
+                    }
+                }
```
Comment (FtheGFW) [inline amigo-lib/src/main/java/me/ele/amigo/utils/PermissionChecker.java]: ArrayList patchPsList = Arrays.asList(patchPs);
ArrayList hostPsList = Arrays.asList(hostPs);
patchPsList.removeAll(hostPsList);

Comment (peacepassion) [inline amigo-lib/src/main/java/me/ele/amigo/utils/PermissionChecker.java]: updated

**Comment by FtheGFW** [comment]
👍

**Pull request merged**